packages/validate/test/ValidateMixin.test.js

```diff
@@ -143,6 +143,100 @@ describe('ValidateMixin', () => {
     expect(otherValidatorSpy.calledWith('foo')).to.equal(true);
   });
 
+  // classes are added only for backward compatibility - they are deprecated
```

Comment:
Do we need tests for attribute reflection?
From the App Developer perspective it makes sense to keep them in specs, since we only reflect a subset of our props, for styling purposes...